group/templates/group/groupTable.html

```diff
@@ -0,0 +1,37 @@
+<table class="table table-hover">
+  <thead>
+    <tr>
+      <th>#</th>
+      <th>Name</th>
+      <th>Owner</th>
+      <th>Group Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for group in group_list %}
+    <tr>
+      <td>{{ group.id }}</td>
+      <td>
+        <a href="{% url 'group:detail' group.id %}">{{ group.gname }}</a>
+      </td>
+      <td>{{ group.owner }}</td>
+      <td>{{ group.description|truncatechars:70 }}</td>
+    </tr>
+    {% endfor %}
+  </tbody>
+</table>
+<div class="container-fluid">
+  <nav>
+    <ul class="pager">
```

Comment:
You can include ```pager.html``` in [utils](https://github.com/bruce3557/NTHUOJ_web/blob/dev/utils/templates/utils/pager.html)